### PR TITLE
GH-1030 Fixes html-color form_Input 3 digit and ""

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith @jbrundage @dtsnell4 - Please Review

Fixes console.log errors with property editor and html-colors being in form of "#f00" or ""

Fixes GH-1030

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage tried to do a fix in widget.js ... if there is a better way of doing this let me know

**Comment by dtsnell4** [comment]
looks fine to me

**Comment by jbrundage** [comment]
This will break when a user tries to enter "transparent" or "rgb(255,255,255)"

File: src/common/Utility.js [hunk 1]
```diff
@@ -7,6 +7,27 @@
...
+                        return;
+                    }
+                    return c+c; 
+                })
+                .join("")
+            ;
+        }
+        return _;
+    }
+
```
Comment (GordonSmith) [inline src/common/Utility.js]: Not needed, should use d3 color helpers:  https://github.com/mbostock/d3/wiki/Colors

**Comment by mlzummo** [comment]
@GordonSmith made suggested changes however .. "transparent" will no longer function as a valid color and im not sure if rgb"a" values will anymore etierh ...

**Comment by GordonSmith** [comment]
@jbrundage Please re-review

**Comment by jbrundage** [comment]
Looks good to me

**Comment by mlzummo** [comment]
fixed issue where it would not work with transparent should work for everything now ...

**Pull request merged**